### DISCUSSION TITLE
random reboots of CerboGX with serial driver

### DISCUSSION THREAD

**Discussion by DirkReudenbach** [Louisvdw/dbus-serialbattery#803] [Q&A] [answered]
I´m running a Victron 3ph ESS System with CerboGX and 4x JKBMS 16s and nodered with some actions

The system runs fine without the serial driver regardless which version. 

If the serial driver is installed (serial, bluetooth or both mixed) the cerbogx starts rebooting randomly - it seems it is going to happen the more load is concerned (but this is just a feeling and could not be judged so far) The reboots will happen several times a day. 

Does anybody have a similar issue and can give some informations - maybe Louis or Mr.Manuel?

**Comment by ramack** [comment]
I have the same issue using 4x Heltec BMS. I tracked it down to the watchdog (configured somewhere in /etc) resetting due to high load.

My solution was to increase the thresholds in the watchdog configuration and adjusting the poll time from 1 to 5 seconds in the driver...


Edit: forgot to say: I'm not on a Cerbo but on a (less powerful) beaglebone.

**Comment by mr-manuel** [comment]
Did you read the whole [How to troubleshoot](https://louisvdw.github.io/dbus-serialbattery/troubleshoot/) page? The more informations you give the more informations we can give you back. Open a [support request](https://github.com/Louisvdw/dbus-serialbattery/issues/new?assignees=&labels=support&projects=&template=support_request.yml), if you need help. Please read carefully which data is needed. Do not forget to pipe the output through `tai64nlocal`, if you don't copy the commands from the documentation.

**Comment by DirkReudenbach** [comment]
Thx Mr-Manuel for your advices. I walked through the troubleshoot page - unfortunately my problem is not described anywhere. The driver overall works fine - the only issue I have is the random reboots several times a day.

**Reply by mr-manuel** [reply]
Check also with `top` command which process is taking the most CPU power. You can exit again with `CTRL + C`.

**Comment by mr-manuel** [comment, marked as answer]
Issue https://github.com/Louisvdw/dbus-serialbattery/issues/808  was opened for this.
